src/Controller/CRUDController.php

```diff
@@ -1563,6 +1569,31 @@ private function checkParentChildAssociation(Request $request, object $object):
         }
     }
 
+    /**
+     * Checks whether $parentObject is equal to $childParentValue or part of it.
+     *
+     * @param object          $needle   Object to compare with $haystack
+     * @param object|iterable $haystack Object to compare with $needle
+     *
+     * @return bool true when $haystack equals $needle or $haystack is iterable and contains $needle
+     */
+    private static function equalsOrInList(object $needle, $haystack): bool
```

Comment:
Even if it doesn't need `$this`, is there any benefit to create it as `static`?

Looking at the method name, what about:

```suggestion
    private function hasParentChildAssociation(object $parent, $parentAssociation): bool
```

Reply:
If this is static, it could be in another class instead of this big CRUDController.